### ISSUE TITLE
No support for other devices

### ISSUE THREAD

**Issue by k2hanson** [Einstein42/myq-garage#14]
I was looking through code and saw no support for light modules. I have one installed at my residence and determined looking at Attributes in the JSON file the device ID is 3. I cobbled it into my code along with the option to use ./myq-garage status [device ID] to check individual device statuses. I am NOT an expert coder so take anything I've done with a grain of salt and try not to laugh....I'm learning as I go....I attached my code.
[myq-garage_rev.txt](https://github.com/Einstein42/myq-garage/files/853055/myq-garage_rev.txt)

**Comment by brbeaird** [comment]
Yes, the light module is ID 3, and you simply send 1 or 0 to turn on or off. I have one working perfectly with the SmartThings integration: https://github.com/brbeaird/SmartThings_MyQ
